demo/bottom_up_video_demo.py

```diff
@@ -0,0 +1,93 @@
+import os
+from argparse import ArgumentParser
+
+import cv2
+
+from mmpose.apis import (inference_bottom_up_pose_model, init_pose_model,
+                         vis_pose_result)
+
+
+def main():
+    """Visualize the demo images."""
+    parser = ArgumentParser()
+    parser.add_argument('pose_config', help='Config file for pose')
+    parser.add_argument('pose_checkpoint', help='Checkpoint file for pose')
+    parser.add_argument('--video-path', type=str, help='Video path')
+    parser.add_argument(
+        '--show',
+        action='store_true',
+        default=False,
+        help='whether to show visualizations.')
+    parser.add_argument(
+        '--out-video-root',
+        default='',
+        help='Root of the output video file. '
+        'Default not saving the visualization video.')
+    parser.add_argument(
+        '--device', default='cuda:0', help='Device used for inference')
+    parser.add_argument(
+        '--kpt-thr', type=float, default=0.3, help='Keypoint score threshold')
+
+    args = parser.parse_args()
+
+    skeleton = [[16, 14], [14, 12], [17, 15], [15, 13], [12, 13], [6, 12],
+                [7, 13], [6, 7], [6, 8], [7, 9], [8, 10], [9, 11], [2, 3],
+                [1, 2], [1, 3], [2, 4], [3, 5], [4, 6], [5, 7]]
+
+    assert args.show or (args.out_video_root != '')
+    assert 'cuda' in args.device
```

Comment:
what about the cpu version?

Reply:
We have not tested it on cpu yet. Not sure if it will work..

Reply:
Support cpu version now.